### PR TITLE
add HAKCRCF411D

### PR DESCRIPTION
**Pull request by WilsonChang**
Please pull and add new unified target HAKCRCF411D, thanks.

### CODE REVIEW

File: configs/default/HARC-HAKRCF411D.config [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+set battery_meter = ADC
+set beeper_inversion = ON
+set beeper_od = OFF
+set sdcard_detect_inverted = OFF
+set sdcard_mode = SPI
+set sdcard_dma = OFF
+set sdcard_spi_bus = 2
+set sdio_clk_bypass = OFF
+set sdio_use_cache = OFF
+set sdio_use_4bit_width = OFF
```
Comment (DusKing1) [inline configs/default/HARC-HAKRCF411D.config:116]: If you are using `SPIFLASH` as your blackbox device, why keep those SDcard configs?

File: configs/default/HARC-HAKRCF411D.config [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set led_inversion = 0
+set dashboard_i2c_bus = 1
+set dashboard_i2c_addr = 60
```
[the remark this answers is not in the document]
Comment (WilsonChang) [reply configs/default/HARC-HAKRCF411D.config:121]: I will check with factory and modify soon, thanks.

[the remark this answers is not in the document]
Comment (WilsonChang) [reply configs/default/HARC-HAKRCF411D.config]: I already deleted unnecessary sets and factory tested, thank you.

**Review by haslinghuis — APPROVED** [review]

**Comment by WilsonChang** [comment]
No changes to HAKRCF411D since last approval, please check and approve again.

**Comment by WilsonChang** [comment]
pull request later.

**Pull request closed without merging**